.pre-commit-config.yaml

```diff
@@ -8,12 +8,14 @@ repos:
     hooks:
     - id: black
       language_version: python3.8
+      additional_dependencies:
+      - click<8.1
 -   repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.4
     hooks:
         - id: flake8
 -   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.812
+    rev: v0.971
```

Comment:
Addresses https://github.com/python/mypy/issues/5887 for `scrapy_zyte_api.responses.ZyteAPIMixin`.